### ISSUE TITLE
Component "postHandle" Method Does Not Receive Message The 2nd Time Sent

### ISSUE THREAD

**Issue by smgonzales** [JacpFX/JacpFX#47]
Hi,

I ran into an issue with 2.1-RC4 where a component is sending a message within the same component from the handle method.  The first time the component's postHandle method received the message, but the second time the component sent the same message the postHandle no longer received the message.  I have zipped up sample, but I'm not able to attach it to this issue.  I can provide the sample to you by whatever means you'd like.

Reproduction Steps:

1) Run sample.
2) Press "Go To Component TWO" button.
3) See component two is displayed.
4) Press "Go To Component ONE" button.
5) See component one is displayed again like the first time in step #1.
6) Press "Go To Component TWO" button again and notice component two is not displayed like in step #3.
7) The message sent by the handle method of component two is never received by postHandle.

Thanks

Santiago

**Comment by amoAHCP** [comment]
Hi,
can you provide a link to GoogleDrive or so?

Thank you

Andy

**Comment by smgonzales** [comment]
Hi Andy,

Here is the sample.

https://drive.google.com/file/d/0B99ZSYhBwXOgNVdhZk5kcm4zbm8/view?usp=sharing_eid&invite=CPnyvdsM&ts=5638ce31

This may not be an issue.  I guess it depends upon if sending a message to a component should once again make that component active again or if you explicitly have to do it yourself once it's already been active once.  The sample will show you what I'm doing.

Thanks

Santiago

**Comment by amoAHCP** [comment]
Hi Santiago, I took a look at your sample (nice to see that it is running as a simple IntelliJ project... havn't used it like this). The problem in your example is the context.active(false)... this is meant to be used when you really want to disable your component (shutdown)... I'll close this issue if it's ok for you.

Andy

**Comment by smgonzales** [comment]
Thanks Andy, for looking at the sample.  In my case I do want to shutdown the component and then later activate it again.  I found that the "handle" method in the FXComponent still receives message so I'm able to set the component to active again which then allows the "postHandle" to receive messages and visibly display (set visible) the component.  

I haven't had any other issues and your framework is at the core of my code.  I particularly like the Callback components.  They've been very useful.

If I had JacpFX wish list I would be able to create a Fragment within a Fragment (so basically a fragment that then loads other fragments).  I have a component that is a Fragment that contains other fragments, but I can only build/construct it at the Perspective and FXComponent level to put the pieces together (popup FXML Dialog Pane containing other Pane Containers where each pane container data is aggregated by the top Dialog Pane).  Other than that I've been able to do everything I've tried.

Santiago
